### PR TITLE
fix: upgradehandler agoric-upgrade-8 init in the right order

### PR DESCRIPTION
**Pull request by arirubinstein**
Fixes the upgrade order, and genesis params for swingset init

### CODE REVIEW

**Comment by arirubinstein** [comment]
Tested on loadtest successfully - ready for review.

File: golang/cosmos/app/app.go [hunk 1]
```diff
@@ -767,38 +771,26 @@ func NewAgoricApp(
...
+			swingsettypes.DefaultBootstrapVatConfig = "@agoric/vats/decentral-test-psm-config.json"
 		default:
-			return vm, fmt.Errorf("invalid upgrade name")
+			return fromVm, fmt.Errorf("invalid upgrade name")
 		}
-		ctx.Logger().Info("post-swingset upgrade", "subspace", swingset.ModuleName, "params", currentSwingsetParams)
 
-		app.SwingSetKeeper.SetParams(ctx, currentSwingsetParams)
+		//Run migrations so InitGenesis is called for lien, swingset, vibc, vbank, vstorage
+		fromVm, err := app.mm.RunMigrations(ctx, app.configurator, fromVm)
```
Comment (JimLarson) [inline golang/cosmos/app/app.go:788]: Since you do the same thing in both cases, can shorten this to

```
return app.mm.RunMigrations(ctx, app.configurator, fromVm)
```

File: golang/cosmos/x/swingset/types/default-params.go [hunk 2]
```diff
@@ -71,3 +60,17 @@ var (
...
+		NewStringBeans(BeansPerBlockComputeLimit, DefaultBeansPerBlockComputeLimit),
+		NewStringBeans(BeansPerFeeUnit, DefaultBeansPerFeeUnit),
+		NewStringBeans(BeansPerInboundTx, DefaultBeansPerInboundTx),
+		NewStringBeans(BeansPerMessage, DefaultBeansPerMessage),
+		NewStringBeans(BeansPerMessageByte, DefaultBeansPerMessageByte),
+		NewStringBeans(BeansPerMinFeeDebit, DefaultBeansPerMinFeeDebit),
+		NewStringBeans(BeansPerVatCreation, DefaultBeansPerVatCreation),
+		NewStringBeans(BeansPerXsnapComputron, DefaultBeansPerXsnapComputron),
+	}
+}
```
Comment (JimLarson) [inline golang/cosmos/x/swingset/types/default-params.go:76]: Add newline at EOF.

**Review by JimLarson — APPROVED** [review]
LGTM with minor fixes.
